packages/webdriverio/src/commands/browser/switchWindow.js

```diff
@@ -15,7 +15,7 @@
         browser.switchWindow('google.com')
 
         // switch back via title match
-        browser.switchWindow('WebDriver test framework')
+        browser.switchWindow('WebDriver bindings for Node.js')
```

Comment:
Since these are the docs for the beta page we should use `Next-gen WebDriver test framework` from http://beta.webdriver.io/